docs/connector/08-01-establish-secure-connection-with-compass.md

```diff
@@ -55,7 +55,8 @@ To establish a secure connection with Compass and generate the client certificat
 
 3. Generate a key and a Certificate Signing Request (CSR).
 
-    Generate a CSR with the following command. `SUBJECT` is the certificate subject data returned with the CSR information as `subject`.   
+    Generate a CSR with the following command. `SUBJECT` is the certificate subject data returned with the CSR information as `subject`. 
+    For MacOS use a forward slash as a delimiter between the subject attributes (e.g. "/O=Org/OU=OrgUnit/L=locality/ST=province/C=DE/CN=54e99a")
```

Comment:
Minor: could you replace the CN value with a placeholder.

For example: `CN=<value>` or something in that direction.

Reply:
Done, thanks.